src/irbasis3/poly.py

```diff
@@ -87,6 +88,50 @@ def value(self, l, x):
         res *= self._norm[i]
         return res
 
+    def overlap(self, f, deg=None, axis=None):
+        """
+        Evaluate overlap \int dx p(x) f(x) using piecewise Gauss-Legendre quadrature,
+        where $p(x)$ are the polynomials.
+
+        f: funtion-like object that evalues f(x) for a 1D array of x.
+           The first axis of the result must correspond to x axis.
```

Comment:
Here, the docs for the `f` argument and `axis` seems to be contradictory. `f` specifies `x` shall be the first dimension, while `axis` defaults to the last axis.

Reply:
Thank you. I will fix this typo!